rust/worker/src/execution/operators/knn_hnsw.rs

```diff
@@ -17,9 +17,10 @@ use super::knn::{KnnOperator, RecordDistance};
 #[derive(Debug)]
 pub struct KnnHnswInput {
     pub hnsw_provider: HnswIndexProvider,
-    pub collection: Collection,
     pub hnsw_segment: Segment,
+    pub collection_dimension: u32,
     pub compact_offset_ids: SignedRoaringBitmap,
+    pub metric: DistanceFunction,
```

Comment:
why introduce new words? just call it a distance_function

Reply:
Fixed.